### DISCUSSION TITLE
How to change bandwidth?

### DISCUSSION THREAD

**Discussion by AnVuNam** [open-sdr/openwifi#406] [Q&A] [answered]
Hi, I see in the [feature](https://github.com/open-sdr/openwifi) it says "Easy to change bandwidth and frequency". However, when reading the [Openwifi documentation](https://github.com/open-sdr/openwifi/blob/master/doc/README.md#let-openwifi-work-at-arbitrary-frequency), I did not see any mention of how to change the bandwidth.

Is there any documentation regarding this? More specifically, if I want to change the bandwidth to 2MHz or 5MHz, how should I do it?

**Comment by JiaoXianjun** [comment, marked as answer]
Please search in issues. or open an issue based on template.
